addons/common/XEH_preStart.sqf

```diff
@@ -15,3 +15,5 @@ uiNamespace setVariable [QGVAR(addonCache), createHashMap];
 
 // Cache for FUNC(getConfigName)
 uiNamespace setVariable [QGVAR(configNames), createHashMap];
+
+uiNamespace setVariable [QGVAR(wheelSelections), createHashMap];
```

Comment:
ok to move to just missionNamespace (preInit)?

Reply:
Sure.